packages/http/src/mocker/__tests__/HttpMocker.spec.ts

```diff
@@ -229,6 +228,118 @@ describe('HttpMocker', () => {
             expect(response.body).toEqual((selectedExample as INodeExample).value);
           });
         });
+
+        describe('and a response example is defined, but no response example is requested', () => {
+          it('returns the first example', () => {
+            const response = httpMocker.mock({
+              input: mockInput,
+              resource: mockResource,
+              config: { mock: { dynamic: false } },
+            });
+
+            expect(response.body).toBeDefined();
+
+            const selectedExample = flatMap(mockResource.responses, res =>
+              flatMap(res.contents, content => content.examples || []),
+            )[0];
+
+            expect(selectedExample).toBeDefined();
+            expect(response.body).toEqual((selectedExample as INodeExample).value);
+          });
+        });
+
+        describe('and the resource has no response examples', () => {
+          const resourceOperation: IHttpOperation = {
```

Comment:
Ok I see what you did here, but that object seems to be too big.
Wouldn't it make better less coupled tests if you split it into smaller, more targeted resource operations?

Reply:
I need you to elaborate more, I fail to understand what you mean.

Reply:
Ah ok maybe you mean to create smaller schemas with the single properties and test them accordingly, right?

Reply:
7123262ed82371f969c4af2a8d98e4ccfa3af588